workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Solutions/NewSolutionFlow/pages/Profile.tsx

```diff
@@ -3,117 +3,253 @@
+							<a
+								href="https://www.liferay.com/trademark"
+								target="_blank"
+							>
+								trademark policy
+							</a>{' '}
+							for details.
+						</span>
+					)}
+					uploadedFile={file}
+				/>
 			</div>
 
-			<Form.Label className="mt-5" htmlFor="name" required>
+			<Form.Label
+				className="mt-5"
+				htmlFor="name"
+				info={tooltipInfo.name}
+				required
+			>
 				{i18n.translate('name')}
 			</Form.Label>
 
 			<Form.Input
 				name="name"
+				onChange={onChange}
 				placeholder="Enter solution name"
 				type="text"
 			/>
 
-			<Form.Label className="mt-5" htmlFor="description" required>
+			<Form.Label
+				className="mt-5"
+				htmlFor="description"
+				info={tooltipInfo.description}
+				required
+			>
 				{i18n.translate('description')}
 			</Form.Label>
 
 			<Form.Input
 				component="textarea"
 				name="description"
+				onChange={onChange}
 				placeholder="Enter solution description"
 				type="textarea"
 			/>
 
-			<div className="form-multiselect">
-				<Form.Label className="mt-5" htmlFor="categories" required>
-					{i18n.translate('categories')}
-				</Form.Label>
-
-				<ClayMultiSelect
-					inputName="description-selector"
-					items={items}
-					onChange={setValue}
-					onItemsChange={setItems}
-					sourceItems={sourceItems}
-					spritemap={getIconSpriteMap()}
-					value={value}
-				/>
+			{!isLoading && (
+				<div className="form-multiselect">
+					<Form.Label
+						className="mt-5"
+						htmlFor="categories"
+						info={tooltipInfo.categories}
+						required
+					>
+						{i18n.translate('categories')}
+					</Form.Label>
 
-				<Form.Label className="mt-5" htmlFor="tags" required>
-					{i18n.translate('tags')}
-				</Form.Label>
-
-				<ClayMultiSelect
-					inputName="tags-selector"
-					items={items}
-					onChange={setValue}
-					onItemsChange={setItems}
-					sourceItems={sourceItems}
-					spritemap={getIconSpriteMap()}
-					value={value}
-				/>
-			</div>
+					<ClayMultiSelect
+						{...{placeholder: 'Select categories'}}
+						inputName="description-selector"
+						items={categories}
+						key={'cat-' + categories.length}
+						onChange={(value: string) =>
+							onChangeMultiSelect({
+								target: {
+									name: 'categories',
+									value,
+								},
+							})
+						}
+						onItemsChange={(value: {[key: string]: string}[]) =>
+							onChange({
+								target: {name: 'categories', value},
+							})
+						}
+						sourceItems={getFilteredItems(
+							categories,
+							defaultSourceItems?.categories
+						)}
+						spritemap={getIconSpriteMap()}
+						value={multiSelectText?.categories}
+					/>
+
+					<Form.Label
+						className="mt-5"
+						htmlFor="tags"
+						info={tooltipInfo.tags}
+						required
+					>
+						{i18n.translate('tags')}
+					</Form.Label>
+
+					<ClayMultiSelect
+						{...{placeholder: 'Select tags'}}
+						inputName="tags-selector"
+						items={tags}
+						key={'tags-' + tags.length}
```

Comment:
Could you use template string on this
```suggestion
						items={tags}
						key={`tags-${tags.length}`}
```